triggers/test/daml/Tests/Matching.daml

```diff
@@ -56,13 +66,14 @@ runMatchingTest parties@Parties{userAdmin = operator; ..} = do
     observers = []
   createListingRequestCid <- alice.customer `submit` do exerciseCmd listingServiceCid Listing.RequestCreateListing with ..
 
-  debug "Waiting for listing to be accepted..."
+  -- debug "Waiting for listing to be accepted..."
```

Comment:
Can either decomment or delete this one (since it already has the "Creating listing..." message earlier)